### PR TITLE
[image-picker][Android] Add Glide for loading images into bitmap

### LINKED ISSUE #3887: [expo-image-manipulator][Android] Crop fails on SDK32
### Environment

Expo SDK32 on Android

### Steps to Reproduce

Launch `ImageManipulator.manipulate` with cropping operation

### Expected Behavior

Crop works just fine and returns valid cropped image.

### Actual Behavior

Crop fails with exception `[Unhandled promise rejection: Error: Invalid crop options has been passed. Please make sure the requested crop rectangle is inside source image.]`

### Reproducible Demo

https://snack.expo.io/@barthec/android-camera-bug---2

### Side notes

Since SDK32 on Android we use `Fresco` library to handle loading images.
These lines are the cause.
https://github.com/expo/expo/blob/master/android/versioned-abis/expoview-abi32_0_0/src/main/java/abi32_0_0/host/exp/exponent/modules/api/ImageManipulatorModule.java#L58-L62

### PR DESCRIPTION
**Pull request by bbarthec**
# Why

Resolves #3887
- `Android`: `Fresco` library would not load images larger than 2048x2048px 
See https://frescolib.org/docs/troubleshooting.html with heading `Bitmap too large to be uploaded to a texture`

# How

- `Android`: I've added new library ([Glide](https://github.com/bumptech/glide)) for handling loading images into Bitmap

# Test Plan

Snack: https://snack.expo.io/@barthec/Z2l0aH

### CODE REVIEW

**Comment by sjchmiela** [comment]
Do you think it would make sense to instead of adding Glide dependency to `expo-image-picker`, add it to `@unimodules/react-native-adapter` instead + adding a new method to `unimodules-image-loader-interface` so there are eg.:
- `loadImageForManipulation` — (naming may change) — that would load a full image to the memory, (without caching?)
- `loadImageForDisplay` — loads image with Fresco or other platform-specific (RN, Flutter) library that is already configured to cache images in some platform-specific-efficient way.

File: packages/expo-image-manipulator/android/src/main/java/expo/modules/imagemanipulator/ImageManipulatorModule.java [hunk 1]
```diff
@@ -69,25 +68,15 @@ public void manipulateAsync(final String uri, final ArrayList<Object> actions, f
...
-              basicMessage + ": " + cause.toString(), cause);
-        } else {
-          promise.reject(ERROR_TAG + "_DECODE", basicMessage + ".");
-        }
-      }
-    });
+    Glide.with(getContext())
+        .asBitmap()
+        .load(uri)
+        .into(new SimpleTarget<Bitmap>() {
```
Comment (sjchmiela) [inline packages/expo-image-manipulator/android/src/main/java/expo/modules/imagemanipulator/ImageManipulatorModule.java]: Will this while invocation cache the image? Do we want to cache the images with ImageManipulator?

**Review by sjchmiela — COMMENTED** [review]
Thanks for taking care of this, left some thoughts

File: packages/expo-image-manipulator/android/build.gradle [hunk 2]
```diff
@@ -66,5 +66,6 @@ dependencies {
   unimodule 'unimodules-core'
   unimodule 'unimodules-image-loader-interface'
 
-  api 'com.android.support:support-annotations:27.1.1'
```
Comment (mczernek) [inline packages/expo-image-manipulator/android/build.gradle]: Compile, target and major of support should match. Se we should wither bump sdk to 28, or leave support annotations as they are.

**Review by mczernek — APPROVED** [review]

File: android/app/proguard-rules.pro [hunk 3]
```diff
@@ -157,3 +157,11 @@
 -keep class org.spongycastle.**
 -dontwarn org.spongycastle.jce.provider.X509LDAPCertStoreSpi
 -dontwarn org.spongycastle.x509.util.LDAPStoreHelper
+
+##### Glide #####
+-keep public class * implements com.bumptech.glide.module.GlideModule
```
Comment (sjchmiela) [inline android/app/proguard-rules.pro]: Was it necessary to add these instructions?

**Review by sjchmiela — APPROVED** [review]
Thanks, LGTM! 💯

File: android/app/proguard-rules.pro (hunk 3 above)
Comment (bbarthec) [reply android/app/proguard-rules.pro]: It turned out that these are not needed 😁

**Pull request merged**